### PR TITLE
Scripts to install app from dir

### PR DESCRIPTION
**Pull request by 1000TurquoisePogs**
Trimmed down version of https://github.com/zowe/zlux-app-server/pull/35 just for installing, without packaging and unpackaging. See there for description.

### CODE REVIEW

File: bin/install-app.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# 
+# SPDX-License-Identifier: EPL-2.0
+# 
+# Copyright Contributors to the Zowe Project.
+if [ $# -eq 0 ]
+    then
+    echo "Usage: $0 AppPath"
+    exit 1
+fi
+utils_path=$(pwd)/../../zlux-server-framework/utils
```
Comment (fkovinAtRocket) [inline bin/install-app.sh]: This requires that the current directory is zlux-app-server/bin.

What is someone runs it differently, e.g. they add  zlux-app-server/bin to PATH and just say install-app.sh? Or they cd into the plugin dir and say $ZOWE_APP_SERVER/bin/install-app.sh? Do we want to support it?

File: bin/install-app.sh [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+if [ "$1" = "/*" ]
```
Comment (fkovinAtRocket) [inline bin/install-app.sh]: Could you please explain what this means?

File: bin/install-app.sh (hunk 1 above)
Comment (1000TurquoisePogs) [reply bin/install-app.sh]: We do want to support it, I just didn't yet. Do you have a suggestion? I'm still learning shell scripting.

File: bin/install-app.sh (hunk 2 above)
Comment (1000TurquoisePogs) [reply bin/install-app.sh]: I believe this was to detect relative versus absolute paths.

Comment (fkovinAtRocket) [reply bin/install-app.sh]: Well, I don't think it works that way. Relative path is always relative to $(pwd) anyway, so no need to do that unless you later cd

**Pull request merged**